spec/requests/api/user_can_search_for_a_song_to_post_spec.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.describe 'GET /api/tracks', type: :request do
+  describe 'successfully get a list of songs' do
+    before do
+      get '/api/tracks', params: { q: 'All I Want for Christmas Is You' }
```

Comment:
You want to set up WebMock so you avoid making the call in test env.

Reply:
This seems like as good a time as any to check my account details and let you know how many calls we can make for when we start with dry runs

Reply:
> This seems like as good a time as any to check my account details and let you know how many calls we can make for when we start with dry runs

What do you mean?

Reply:
> You want to set up WebMock so you avoid making the call in test env.

We fixed that. Thanks.

Reply:
@tochman We set up Webmock👍